src/stores/authStore.ts

```diff
@@ -0,0 +1,126 @@
+import { create } from 'zustand';
+import { persist } from 'zustand/middleware';
+
+// 사용자 정보 타입
+export interface UserInfo {
+  id: number;
+  username: string;
+  email: string;
+  nickname: string;
+  profileImageUrl: string;
+}
+
+export interface SignInUser {
+  accessToken: string;
+  user: {
+    id: number;
+    username: string;
+    email: string;
+    nickname: string;
+    profileImageUrl: string;
+  };
+}
```

Comment:
완전히 내용이 일치해서 이렇게 수정해도 될것 같아용
```suggestion
export interface SignInUser {
  accessToken: string;
  user: UserInfo;
}
```